src/i_video.c

```diff
@@ -420,28 +433,62 @@ static inline void I_UpdateRender (void)
     }
 }
 
+#define DRS_MIN_HEIGHT 400
+#define DRS_STEP       1
+#define DRS_MIN_DELTA  10
+static int drs_percent = 100;
+static boolean drs_update;
```

Comment:
@fabiangreffrath  With the latest changes, the "vibration" has been significantly reduced. I'm not sure if it can be removed completely, it appears even with integer multipliers. But I think that with a smarter approach to these parameters we can make it almost imperceptible.

Reply:
This is good news! I expect resolution changes to happen not too often during casual play, so a little visual shake won't be much of a problem.